src/containers/MyNdla/Arena/components/PostList.tsx

```diff
@@ -15,26 +15,88 @@ import { GQLArenaPostV2Fragment, GQLArenaTopicByIdV2Query } from "../../../../gr
 
 const StyledOl = styled("ol", {
   base: {
-    listStyle: "none",
-    marginInlineStart: "large",
-    padding: "unset",
-    mobileWide: {
-      marginInlineStart: "3xlarge",
+    marginInlineStart: "xsmall",
+    tablet: {
+      marginInlineStart: "small",
+    },
+    desktop: {
+      marginInlineStart: "medium",
     },
   },
 });
 
 const StyledLi = styled("li", {
   base: {
-    padding: "unset",
+    position: "relative",
+    paddingBlockStart: "xxsmall",
+    paddingInlineStart: "xsmall",
+    borderInlineStart: "1px solid",
+    borderColor: "stroke.subtle",
+    "&:last-of-type": {
+      borderInlineStart: "unset",
+    },
+    "&:not(:last-of-type)": {
+      _after: {
+        borderInlineStart: "unset",
+      },
+    },
+    "&[data-padding='true']": {
+      paddingInlineStart: "none",
+    },
+    _after: {
+      borderInlineStart: "1px solid",
+      borderBlockEnd: "1px solid",
+      borderColor: "stroke.subtle",
+      borderBottomLeftRadius: "xsmall",
+      position: "absolute",
+      content: "''",
+      width: "xsmall",
+      height: "xxlarge",
+      left: "0px",
+      top: "0px",
+    },
+    tablet: {
+      paddingBlockStart: "small",
+      paddingInlineStart: "medium",
+      _after: {
+        width: "large",
+        height: "3xlarge",
+      },
+    },
+    desktop: {
+      paddingBlockStart: "medium",
+      paddingInlineStart: "xxlarge",
+      _after: {
+        width: "xxlarge",
+        height: "3xlarge",
+      },
+    },
+  },
+});
+
+const EditorWrapper = styled("div", {
```

Comment:
Er ikke denne definert et annet sted også?

Reply:
Nope, kan sikkert gi den et annet navn, den er kun for å spacingen riktig i forhold til linjene.